configuration/amd64/mysql/localhost.sql

```diff
@@ -78,7 +78,6 @@ CREATE TABLE `installedPlugin` (
 
 LOCK TABLES `installedPlugin` WRITE;
 /*!40000 ALTER TABLE `installedPlugin` DISABLE KEYS */;
```

Comment:
Should we remove the lock and unlock statements if we are not writing data into the tables?

Reply:
We can remove the lock statements although functionally it would not impact anything. I see these sql files as possible avenues to restoring a specific state of V2X-Hub. Then then these lock statements offer an outline to how to create a state. But since these files are generated normally by mysqldump, and the lock statements will be generated as well I can remove then since they do not serve a purpose.